test/Serilog.Formatting.Compact.Tests/Serilog.Formatting.Compact.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
+    <AssemblyName>Serilog.Formatting.Compact.Tests</AssemblyName>
+    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <IsPackable>false</IsPackable>
```

Comment:
This tag is not necessary.

Reply:
I think this could be beneficial later.
It lets just run dotnet pack on solution level.